free-programming-books-ko.md

```diff
@@ -20,7 +20,7 @@
 ### GIT
 
 * [Pro Git 한글 번역](http://git-scm.com/book/ko/)
-* [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown)
+* [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown/)
```

Comment:
Done in https://github.com/davisonio/awesome-free-programming-books/commit/68ce19c181906803fb4f3bebb239f376a7326d05